src/native/libs/System.Security.Cryptography.Native/pal_x509.c

```diff
@@ -871,27 +871,29 @@ static OCSP_CERTID* MakeCertId(X509* subject, X509* issuer)
     return OCSP_cert_to_id(EVP_sha1(), subject, issuer);
 }
 
+static time_t GetIssuanceWindowStart()
+{
+    // time_t granularity is seconds, so subtract 4 days worth of seconds.
+    // The 4 day policy is based on the CA/Browser Forum Baseline Requirements
+    // (version 1.6.3) section 4.9.10 (On-Line Revocation Checking Requirements)
+    time_t t = time(NULL);
+    t -= 4 * 24 * 60 * 60;
+    return t;
+}
+
 static X509VerifyStatusCode CheckOcsp(OCSP_REQUEST* req,
                                       OCSP_RESPONSE* resp,
                                       X509* subject,
                                       X509* issuer,
                                       X509_STORE_CTX* storeCtx,
-                                      ASN1_GENERALIZEDTIME** thisUpdate,
-                                      ASN1_GENERALIZEDTIME** nextUpdate)
```

Comment:
The cache policy was repeated across two callers, and almost got repeated to a third.  Now it's inside this routine and just expressed via `canCache`... so no one needs these values any more.